package.json

```diff
@@ -9,6 +9,7 @@
   "scripts": {
     "build:analyze": "yarn build --stats-json && npx --yes webpack-bundle-analyzer dist/storefrontapp/stats.json",
     "build": "env-cmd --no-override -e dev,b2c,$SPA_ENV ng build storefrontapp --configuration production",
+    "build:b2b": "env-cmd --no-override -e dev,b2b,$SPA_ENV ng build storefrontapp --configuration production",
```

Comment:
In build, we don't separate b2b or b2c, right? It just build `storefrontapp`.

Reply:
correct

Reply:
builds storefrontapp with b2b features

Reply:
only when run storefrontapp, pass `b2b` make sense. In build, we don't separate b2b or b2c